sw/device/boot_rom2/ext_format.c

```diff
@@ -0,0 +1,130 @@
+// Cn:opyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/boot_rom2/ext_format.h"
+
+#include "sw/device/lib/base/memory.h"
+#include "sw/device/lib/runtime/check.h"
+
+static const uint32_t kExtMagic = 0x5245545f;
```

Comment:
nit: ascii = "RET_" ? Where did this come from? Comment if it makes sense to add context.

Reply:
It's little endian, so the ascii is actually "_TER". THat's actually a typo on my part, too: the first character should be an O, for "OpenTitan Ext Rom".

Good catch.